Gems/GraphModel/Code/Source/Model/Slot.cpp

```diff
@@ -46,6 +46,7 @@ namespace GraphModel
                 ->Attribute(AZ::Script::Attributes::Module, "editor.graph")
                 ->Constructor<const SlotName&>()
                 ->Constructor<const SlotName&, SlotSubId>()
+                ->Method("__repr__", [](SlotId* that) { return that->GetRepr(); })
```

Comment:
Thank you! Since you added the member function, can this be bound like the other functions for consistency? Or renamed ToString?
```suggestion
                ->Method("__repr__", &Slot::ToString)
```